doc/configuration_options.rst

```diff
@@ -81,7 +81,8 @@ A set of options which can be used in any build file.
   The first level key contains the OS name.
   The second level key contains the OS code name.
   The third level key contains the CPU architecture.
-  The third level value is always empty.
+  The third level value is always empty. The first level key "_config" is
```

Comment:
Please start each sentence on a new line (see https://github.com/ros2/ros2/wiki/Developer-Guide#markdown for the rational).

Same below.